CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 ## Next
 
+* Remove deprecated `controllers` field from the worker config, which was deprecated for
```

Comment:
I usually find it's good to mention _when_ it was deprecated, that way if people didn't see the warnings or the deprecation notice they at least have an idea it wasn't a very sudden or recent change.